client/signup/steps/jpo-summary/index.jsx

```diff
@@ -169,19 +206,29 @@ class JPOSummaryStep extends React.Component {
 					</tbody>
 				</table>
 				<div>
-					<Button primary onClick={ this.completeOnboarding }>
+					<a
+						className="jpo-summary__visit-site button is-primary"
+						href={ get( this.props.signupDependencies, [ 'jpoConnect', 'queryObject', 'home_url' ] ) }>
 						{
-							translate( 'Visit your site' )
+							translate( 'Visit site' )
 						}
-					</Button>
+					</a>
 				</div>
 			</div>
 		);
 	}
 
+	componentWillMount() {
+		this.props.updateSettings(
+			get( this.props.signupDependencies, [ 'jpoConnect', 'queryObject', 'client_id' ], -1 ),
+			this.getOnboardingChoices(),
+			false // don't sanitize settings
+		);
+	}
+
 	render() {
-		const headerText = translate( 'Congratulations! %s is on its way.', {
-			args: get( this.props.signupProgress[ 0 ], [ 'jpoSiteTitle', 'siteTitle' ], false ) || translate( 'your new site' )
+		const headerText = translate( 'Congratulations! %s is ready.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Congratulations! %s is available.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).